.github/workflows/rust.yml

```diff
@@ -0,0 +1,44 @@
+name: Rust
+
+on:
+
+  pull_request:
+    branches: [ "master", "20-introduce-ci" ] # We should make this main or mainline.
+    paths-ignore: ['**.md']
```

Comment:
definitely would add an ignore for yml (possibly json, since i see a changed json file in this PR)

Reply:
it might be better to trigger only on *code* changes though, toml or rs only